.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: golang-1.16
+  tag: rhel-8-release-golang-1.17-openshift-4.11
```

Comment:
i am realizing now that this might not be right. this is moving it to the 4.11 version in the 4.10 branch.

Reply:
ah yeah the cherry-pick is wrong :(